### PR TITLE
Publish finetune components

### PR DESCRIPTION
**Pull request by jpmann**
The PR contains 5 NLP tasks - text classification, token classification, summarization, translation and extractive QnA. Each task has 3 command components + 1 pipeline component

### CODE REVIEW

File: training/finetune_acft_hf_nlp/components/finetune/question_answering/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,341 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: question_answering_finetune
+version: 0.0.1
```
Comment (imbalu007) [inline training/finetune_acft_hf_nlp/components/finetune/question_answering/spec.yaml:3]: Please upgrade the version number to match the version number of components in preview registry

**Review by imbalu007 — DISMISSED** [review]

File: training/finetune_acft_hf_nlp/components/finetune/question_answering/spec.yaml (hunk 1 above)
Comment (jpmann) [reply training/finetune_acft_hf_nlp/components/finetune/question_answering/spec.yaml]: updated to 0.0.5

File: training/finetune_acft_hf_nlp/components/preprocess/question_answering/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+name: question_answering_datapreprocess
+version: 0.0.5
+type: command
+
+is_deterministic: True
+
+display_name: Question Answering DataPreProcess
+description: Component to preprocess data for question answering task
+
+environment: azureml:acft-hf-nlp-gpu:2
```
Comment (novaturient95) [inline training/finetune_acft_hf_nlp/components/preprocess/question_answering/spec.yaml:11]: Please use MCR asset URI for all the components.

Comment (jpmann) [reply training/finetune_acft_hf_nlp/components/preprocess/question_answering/spec.yaml]: updated the environment asset URI

**Pull request merged**